pkg/apiserver/apiserver.go

```diff
@@ -289,12 +289,19 @@ func (s *PorchServer) Run(ctx context.Context) error {
 
 	certStorageDir, found := os.LookupEnv("CERT_STORAGE_DIR")
 	if found && strings.TrimSpace(certStorageDir) != "" {
-		if err := setupWebhooks(ctx, webhookNs, certStorageDir); err != nil {
+		// check if we should use cert manager webhook setup
+		useCertManWebhook := false
```

Comment:
We can prob condense this down a bit.

		certManEnvVar, _ := os.LookupEnv("USE_CERT_MANAGER")
		useCertManager, _ := strconv.ParseBool(certManEnvVar)
		if err := setupWebhooks(ctx, webhookNs, certStorageDir, useCertManager); err != nil {

Reply:
- noted. yes i used srtconv way previously but swapped it out for the reason of not wanting to import new packages if i can avoid doing so. i will ammend